source/develop/release-management/features/sla/self-hosted-engine-gluster-support.html.md

```diff
@@ -75,9 +75,9 @@ Like any other gluster storage domain, the hosted engine storage domain on glust
     gluster volume set <volname> group virt
     gluster volume set <volname> storage.owner-uid 36 
     gluster volume set <volname> storage.owner-gid 36
-    gluster volume set <volname> performance.low-prio-threads 32
-    gluster volume set <volname> cluster.data-self-heal-algorithm full
-    gluster volume set <volname> network.ping-timeout   30
+    gluster volume set <volname> performance.strict-o-direct on
+    gluster volume set <volname> network.remote-dio off
+    gluster volume set <volname> network.ping-timeout 30
```

Comment:
Thanks @sabose, having the required configuration in one place is very helpful.

What is missing, is a section for every configuration, explaning why we recommend this option,
and what  are the bad effects of not using them.

The most important settings which needs more info are:
- group virt - where is this documented?
- performance.strict-o-direct on
- network.remote-dio off
- network.ping-timeout 30

network.ping-timeout 30 sounds dangerous, since sanlock uses 10 seconds timeout. I  see in 
one of the duplicate configuration examples bellow that this was 10 seconds in the past. What
was the reason to change this?

Finally, should we enforce these settings in oVirt? When we connect to a storage domain, we get the
volume options, to find the backup servers and the number or bricks. Do you think we should also
verify the options and prevent connecting (or at least warn) about invalid configuration?

Reply:
Thanks! consolidating in 1 place. Regarding network.ping-timeout - for some reason, we have been of the opinion we had to keep this value less than 32 seconds which was the sanlock timeout?
10 seconds is too less as ping-timeout value because this causes frequent brick disconnects. network.ping-timeout is the time duration for which the client waits to check if the server is responsive. The server may not respond either because it's disconnected, or all threads are busy processing i/o requests.
About enforcing these settings - there's no good way to check if a group option has been set on the volume.